samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -145,17 +147,18 @@ private static async Task MemorySampleAsync()
             .Build();
 
         string folder = RepoFiles.SampleSkillsPath();
-        kernel.ImportSemanticSkillFromDirectory(folder, "SummarizeSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "WriterSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "CalendarSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "ChatSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "ChildrensBookSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "ClassificationSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "CodingSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "FunSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "IntentDetectionSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "MiscSkill");
-        kernel.ImportSemanticSkillFromDirectory(folder, "QASkill");
+        kernel.ImportSemanticSkillFromDirectory(folder,
```

Comment:
Things like `ImportSampleSkills` in dotnet\src\SemanticKernel.IntegrationTests\TestHelpers.cs could be improved, too.

Reply:
@lemillermicrosoft Good call. Will add those changes to the PR.